src/v0/destinations/drip/util.js

```diff
@@ -77,18 +82,21 @@ const createUpdateUser = async (finalpayload, Config, basicAuth) => {
         },
       },
       {
+        metadata,
         destType: 'drip',
         feature: 'transformation',
         requestMethod: 'POST',
         endpointPath: '/subscribers',
         module: 'router',
       },
     );
+    const { response } = httpResponse;
     if (response) {
       return response.status === 200 || response.status === 201;
     }
     throw new AbortedError('Invalid response.');
-  } catch (error) {
+  } catch ({ destinationResponse }) {
```

Comment:
Interesting... Are we confident that `destinationResponse` will always be present ?

Reply:
In `myAxios` as soon as we figure out there is an error status-code, we would throw an exception. May be that is the reason we were having a catch block in the first-place.
But now with `handleHttpRequest` method, we wouldn't need to have a catch block may be. Instead we should rely on the response that we got from `handleHttpRequest` & throw an error when required.

May be we should re-think existence of catch block ?